post-scan-actions/aws-python-promote-or-quarantine/other-installation-methods.md

```diff
@@ -366,5 +366,14 @@ Additional options are supported as described in the following section.
 - **Custom ACL (Access Control List)**
     - An access control list (ACL) [defines which AWS accounts or groups are granted access and the type of access](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl_overview.html).
     - You can set a specific object ACL for promoted/quarantined objects. For example, if your source bucket has restricted permissions and you wish to grant wider access to promoted/quarantined objects, set the `ACL` environment variable in the scanner Lambda configuration, i.e., the value `bucket-owner-full-control` grants the promote/quarantine bucket access to the object as if it were there own.
+- **PermissionBoundary**
+    - A [permissions boundary](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html) is an advanced feature for using a managed policy to set the maximum permissions that an identity-based policy can grant to an IAM entity. An entity's permissions boundary allows it to perform only the actions that are allowed by both its identity-based policies and its permissions boundaries.
+    - If you wish to set a permission boundary for the stack, set the `PermissionsBoundary` to the arn of the policy you wish to use.
+- **KMSKeyARN**
+    - [AWS Key Management Service (KMS)](https://aws.amazon.com/kms/) is a managed service that makes it easy for you to create and control the encryption keys used to encrypt your data, and uses FIPS 140-2 validated hardware security modules to protect the security of your keys.
+    - If you wish to use a specific KMS key for the `QUARANTINEBUCKET` and for the `PROMOTEBUCKET`, set the `KMSKeyARN` parameter with the arns of the keys you wish to use.
```

Comment:
```suggestion
    - If you wish to use a specific KMS key for the scanning bucket, `QUARANTINEBUCKET` and for the `PROMOTEBUCKET`, set the `KMSKeyARN` parameter with the arns of the keys you wish to use.
```
Current description sounds like we'll configure SSE for those buckets to me. If you think so as well, maybe we can refer to how our doc describes the parameter:
```
    - Specify the comma delimited ARN list of the KMS master keys used to encrypt S3 bucket objects if you have enabled SSE-KMS for the scanning bucket, quarantined bucket or promote bucket.
```
Ref: `KMSKeyARNForBucketSSE` in https://cloudone.trendmicro.com/docs/file-storage-security/stack-add-aws/.

Reply:
@felipecosta09, would you like to take the public doc's parameter description as the reference?